roles/matrix-bot-maubot/defaults/main.yml

```
@@ -0,0 +1,36 @@
+---
+
+matrix_bot_maubot_enabled: true
+matrix_bot_maubot_container_image_self_build: false
+matrix_bot_maubot_docker_repo: "https://mau.dev/maubot/maubot.git"
+matrix_bot_maubot_docker_src_files_path: "{{ matrix_bot_maubot_base_path }}/docker-src"
+
+matrix_bot_maubot_version: latest
```

`v0.3.1` seems to be available. We'd better use a tagged release, not `latest`.
```suggestion
matrix_bot_maubot_version: v0.3.1
```